metsrw/utils.py

```diff
@@ -13,11 +13,20 @@
 SCHEMA_LOCATIONS = "http://www.loc.gov/METS/ " + \
                    "http://www.loc.gov/standards/mets/version18/mets.xsd"
 
+PREMIS_SCHEMA_LOCATIONS = "info:lc/xmlns/premis-v2 " + \
+                          "http://www.loc.gov/standards/premis/v2/premis-v2-2.xsd"
+
+PREMIS_VERSION = "2.2"
 
 def lxmlns(arg):
     """ Return XPath-usable namespace. """
     return '{' + NAMESPACES[arg] + '}'
 
+def find_element_text(parent_el, path_expression):
+    """ Attempt to find element and, if found, return its text content """
+    el = parent_el.find(path_expression, NAMESPACES)
+    return el.text if el is not None else None
```

Comment:
This is a duplicate of lxml's findtext. See http://lxml.de/tutorial.html#ElementPath and http://lxml.de/api/index.html under `lxml.etree` -> `ElementBase` -> `findtext`